packages/core/src/pages/account/profile.tsx

```diff
@@ -17,14 +18,15 @@ import { default as BeforeSection } from 'src/customizations/src/myAccount/exten
 import type { MyAccountProps } from 'src/experimental/myAccountSeverSideProps'
 import { injectGlobalSections } from 'src/server/cms/global'
 import { getMyAccountRedirect } from 'src/utils/myAccountRedirect'
+import { useCart } from 'src/sdk/cart'
```

Comment:
Are we using `useCart` here?

Reply:
Nope, my editor probably added this by accident. Thanks